frontend/src/resources/rbac.ts

```diff
@@ -84,6 +111,47 @@ export interface ClusterRoleBinding {
   }
 }
 
+export interface Role {
+  apiVersion: RbacApiVersionType
+  kind: RoleKindType
+  metadata: Metadata
+  rules: {
+    verbs: string[]
+    apiGroups: string[]
+    resources: string[]
+    resourceNames: string[]
+  }[]
+}
+
+export interface RoleBinding {
+  apiVersion: RbacApiVersionType
+  kind: RoleBindingKindType
+  metadata: Metadata
+  subjects?: {
+    kind: 'User' | 'Group'
+    apiGroup: 'rbac.authorization.k8s.io'
+    name: string
+    namespace: string
+  }[]
+  roleRef: {
+    apiGroup: 'rbac.authorization.k8s.io'
+    kind: RoleKindType
+    name: string
+  }
+}
+
+export interface ServiceAccount {
+  apiVersion: ServiceAccountApiVersionType
+  kind: ServiceAccountKindType
+  metadata: Metadata
+  secrets: string[]
```

Comment:
the same from `imagePullSecrets` applies to this but using `ObjectReference` from `@openshift-console/dynamic-plugin-sdk` library